.gitconfig

```diff
@@ -1,5 +1,6 @@
 [alias]
 	d = diff -w
+	dt = difftool --dir-diff
```

Comment:
The problem with `--dir-diff` is that we lose information about renames. Try running

```sh
git difftool --tool=meld 36b760f
```

to see how. This will affect any custom diff viewer, too.

Reply:
Let's ignore that loss of information. We don't rename files all the time. Most commits add, change or delete lines.